### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by aoot**
I believe this is what the original author intended as the original statement did not make sense.

### CODE REVIEW

**Review by soosung80 — DISMISSED** [review]
LGTM

**Comment by aoot** [comment]
Apologies, didn't know that additional edits would change the existing PR, thus I removed the additional edits.
The changes is now back to right before I dismissed the stale review.

**Review by 8bitmp3 — APPROVED** [review]
Thanks

File: site/en/guide/intro_to_modules.ipynb [hunk 1]
```diff
@@ -403,7 +403,7 @@
         "id": "4eGaNiQWcK4j"
       },
       "source": [
-        "During distributed (multi-machine) training they can be sharded,  which is why they are numbered (e.g., '00000-of-00001').  In this case, though, there is only have one shard.\n",
+        "During distributed (multi-machine) training they can be sharded,  which is why they are numbered (e.g., '00000-of-00001').  In this case, though, there is only one shared.\n",
```
Comment (8bitmp3) [inline site/en/guide/intro_to_modules.ipynb:406]:
```suggestion
        "During distributed (multi-machine) training they can be sharded,  which is why they are numbered (e.g., '00000-of-00001').  In this case, though, there is only one shard.\n",
```

**Review by 8bitmp3 — APPROVED** [review]

**Pull request merged**